### DISCUSSION TITLE
Unable to install node_modules

### DISCUSSION THREAD

**Discussion by mariaafahad** [nrwl/nx#14272] [General]
**I am trying to install node_modules using yarn install/npm i but failed to install and throwing this error:**

`error @angular-devkit/core@15.0.4: The engine "node" is incompatible with this module. Expected version "^14.20.0 || ^16.13.0 || >=18.10.0". Got "18.9.0"`

@vsavkin please help me.

**Comment by marko-hologram** [comment]
> `error @angular-devkit/core@15.0.4: The engine "node" is incompatible with this module. Expected version "^14.20.0 || ^16.13.0 || >=18.10.0". Got "18.9.0"`

This error says it all. You are running this command with an incompatible Node.js version. Please install one of the defined versions. In your case, since you already have 18.9.0 it would be best to just update to latest 18.x (or at least 18.10.0) and then just run that same command with that newer Node.js version.
